.github/workflows/gateway-conformance.yml

```diff
@@ -0,0 +1,138 @@
+name: Gateway Conformance Dashboard
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  # Enable this when we're ready to generate the dashboard
+  # schedule:
+  #   - cron: "0 */6 * * *" # every six hours
+
+concurrency:
+  group: "conformance"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  configure:
+    runs-on: ubuntu-latest
+    outputs:
+      gateways: ${{ steps.set-matrix.outputs.gateways }}
+    steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+     - name: Set matrix data
+       id: set-matrix
+       run: |
+        jq --raw-input --slurp --compact-output 'split("\n")[:-1]' gateways.txt | \
+          tr -d '\n' | \
+          xargs --null --max-chars=2000000 -I {} echo "gateways={}" | \
+          tee -a "$GITHUB_OUTPUT"
+  conformance:
+    runs-on: ubuntu-latest
+    needs: configure
+    strategy:
+      matrix:
+          gateway_url: ${{ fromJson(needs.configure.outputs.gateways) }}
+      fail-fast: false
+    steps:
+      # 1. Generate the slug used for reporting
+      - name: Generate slug
+        id: slug
+        env:
+          GATEWAY_URL: ${{ matrix.gateway_url }}
+        run: |
+          slug=$(echo "${GATEWAY_URL}" |
+              sed -e 's/http[s]\?:\/\///' \
+              -e 's/[:/@.]/-/g' \
+              -e 's/[^A-Za-z0-9\-]/-/g' |
+              tr "[:upper:]" "[:lower:]")
+          echo "slug=$slug" >> $GITHUB_OUTPUT
+
+      # 2. Run the gateway-conformance tests
+      - name: Run gateway-conformance tests
+        uses: ipfs/gateway-conformance/.github/actions/test@v0
+        with:
+          gateway-url: ${{ matrix.gateway_url }}
+          json: output.json
+          xml: output.xml
+          html: output.html
+          markdown: output.md
+          report: report.json
+          accept-test-failure: true
+
+      # 3. Upload the results
+      - name: Upload MD summary
+        # TODO: generate a minimal output.md in the action
```

Comment:
@laurentsenta to reference issue tracking this work (for dx)

Reply:
```suggestion
        # TODO: generate a minimal output from the action - https://github.com/ipfs/gateway-conformance/issues/171
```